pkg/option/config.go

```diff
@@ -1963,6 +1966,9 @@ type DaemonConfig struct {
 	// IPAM is the IPAM method to use
 	IPAM string
 
+	// Enable chaining with another CNI plugin.
+	CNIChainingMode string
```

Comment:
should we register a flag for this and do `option.BindEnv` in `initializeFlags`?